contracts/contracts/strategies/BridgedWOETHStrategy.sol

```diff
@@ -0,0 +1,185 @@
+import { AggregatorV3Interface } from "../interfaces/chainlink/AggregatorV3Interface.sol";
+import { StableMath } from "../utils/StableMath.sol";
+
+contract BridgedWOETHStrategy is InitializableAbstractStrategy {
+    using StableMath for uint256;
+
+    IWETH9 public immutable weth;
+    IERC20 public immutable bridgedWOETH;
+
+    // TODO: Should this strategy also use Oracle Router?
+    AggregatorV3Interface public immutable oracleFeed;
+    uint8 public immutable oracleFeedDecimals;
+
+    uint256 public constant MAX_PRICE_STALENESS = 2 days;
+
+    /**
+     * @dev Verifies that the caller is the Governor or Strategist.
+     */
+    modifier onlyGovernorOrStrategist() {
+        require(
+            isGovernor() || msg.sender == IVault(vaultAddress).strategistAddr(),
+            "Caller is not the Strategist or Governor"
+        );
+        _;
+    }
+
+    constructor(
+        BaseStrategyConfig memory _stratConfig,
+        address _weth,
+        address _bridgedWOETH,
+        address _oracleFeed
+    ) InitializableAbstractStrategy(_stratConfig) {
+        weth = IWETH9(_weth);
+        bridgedWOETH = IERC20(_bridgedWOETH);
+
+        oracleFeed = AggregatorV3Interface(_oracleFeed);
+        oracleFeedDecimals = AggregatorV3Interface(_oracleFeed).decimals();
+    }
+
+    function initialize() external onlyGovernor initializer {
+        InitializableAbstractStrategy._initialize(
+            new address[](0), // No reward tokens
+            new address[](0), // No assets
+            new address[](0) // No pTokens
+        );
+    }
+
+    function getBridgedWOETHValue(uint256 amount)
+        public
+        view
+        returns (uint256 oethValue)
+    {
+        // slither-disable-next-line unused-return
+        (, int256 oraclePrice, , uint256 updatedAt, ) = AggregatorV3Interface(
+            oracleFeed
+        ).latestRoundData();
+
+        require(
+            updatedAt + MAX_PRICE_STALENESS >= block.timestamp,
+            "Oracle price too old"
+        );
+
+        // Price per unit bridged wOETH
+        oethValue = uint256(oraclePrice).scaleBy(18, oracleFeedDecimals);
+
+        // Price for `amount`
+        oethValue = (amount * oethValue) / 1 ether;
+    }
+
+    // TODO: Add minAmount
+    function depositBridgedWOETH(uint256 woethAmount)
+        external
+        onlyGovernorOrStrategist
+    {
+        // Transfer in all bridged wOETH tokens
+        bridgedWOETH.transferFrom(msg.sender, address(this), woethAmount);
+
+        // Figure out how much they are worth
+        uint256 oethToMint = getBridgedWOETHValue(woethAmount);
+
+        // There's no pToken, however, it just uses WOETH address in the event
+        emit Deposit(address(weth), address(bridgedWOETH), oethToMint);
+
+        // Mint OETHb tokens and transfer it to the caller
+        IVault(vaultAddress).mintToForStrategy(msg.sender, oethToMint);
+    }
+
+    function withdrawBridgedWOETH(uint256 woethAmount, address receiver)
```

Comment:
⚪ It's odd to have the withdraw function take an explicit receiver on where to send funds too, while the deposit function (which also sends funds) does not. I'd probably lean towards removing it from the withdraw function unless there's a need for it.